src/Debug/TraceableTransformer.php

```diff
@@ -105,10 +107,13 @@ public function transform(
         }
 
         try {
-            $start = microtime(true);
+            $id = substr(hash('sha256', uniqid((string) mt_rand(), true)), 0, 6);
```

Comment:
Thanks. I decided to go with xxh128 instead because Symfony uses it everywhere, including in other data collectors. https://github.com/rekalogika/mapper/pull/184

Reply:
But you only use the first 6 characters anyway. What's the advantage of a 128-bit hash over a 64-bit hash in this case?